### DISCUSSION TITLE
Portable Windows version

### DISCUSSION THREAD

**Discussion by eadmaster** [cnr-isti-vclab/meshlab#1493] [Ideas]
is there a portable build for windows of Meshlab? (portable like https://portableapps.com/ )

I've found the `*-windows.zip` releases are still storing the settings in  `HKEY_CURRENT_USER\SOFTWARE\VCG\MeshLab_64bit_fp`.

**Comment by alemuntoni** [comment]
<s>In the [Releases](https://github.com/cnr-isti-vclab/meshlab/releases) page, the `.zip` archive is a portable version.</s>

Sorry, I misread the email.
Right now there is no such portable build. PRs for this are welcome! :)
